From e1f48323a1eb73018e3929eeac65b26eee7207ea Mon Sep 17 00:00:00 2001
From: Jamie Lennox <jamielennox@gmail.com>
Date: Wed, 4 Dec 2019 21:56:45 +1100
Subject: [PATCH] Use system urllib3 package

The fedora requests package uses the system urllib3 rather than the one
bundled into requests. requests-mock must therefore do the same thing.
---
 requests_mock/response.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requests_mock/response.py b/requests_mock/response.py
index d081cf5..13ab2b8 100644
--- a/requests_mock/response.py
+++ b/requests_mock/response.py
@@ -16,9 +16,9 @@ from requests.adapters import HTTPAdapter
 from requests.cookies import MockRequest, MockResponse
 from requests.cookies import RequestsCookieJar
 from requests.cookies import merge_cookies, cookiejar_from_dict
-from requests.packages.urllib3.response import HTTPResponse
 from requests.utils import get_encoding_from_headers
 import six
+from urllib3.response import HTTPResponse

 from requests_mock import compat
 from requests_mock import exceptions
--
2.21.0

